lib/rules/jsx-sort-props.js

```diff
@@ -310,6 +324,27 @@ function validateReservedFirstConfig(context, reservedFirst) {
   }
 }
 
+/**
+ * Checks if the `customPropsFirst` option is valid
+ * @param {Object} context The context of the rule
+ * @param {boolean | string[]} customPropsFirst The `customPropsFirst` option
+ * @return {Function | undefined} If an error is detected, a function to generate the error message, otherwise, `undefined`
+ */
+// eslint-disable-next-line consistent-return
+function validateCustomPropsFirstConfig(context, customPropsFirst) {
+  if (customPropsFirst) {
+    if (Array.isArray(customPropsFirst)) {
+      if (customPropsFirst.length === 0) {
+        return function Report(decl) {
+          report(context, messages.customPropsListIsEmpty, 'customPropsListIsEmpty', {
+            node: decl,
+          });
+        };
+      }
+    }
+  }
+}
```

Comment:
this should be handled by the schema, not by the plugin, and can be removed. (also an empty list is fine)